### DISCUSSION TITLE
Need help in contributing

### DISCUSSION THREAD

**Discussion by vishwadeepthi** [nammayatri/nammayatri#761] [General]
I wanted to take a moment to express my appreciation for your decision to make this open source. I am truly grateful for the opportunity to explore the codebase and learn from your approach to app development. It's evident that you've put a lot of effort and dedication into creating the app. I'm excited to see how the app will continue to evolve with the help of the community.

Am trying to setup the Frontend app locally, and got stuck at the step of adding google-services.json file to app.

From where can i get this file?

Thanks in advance

**Comment by wololowarrior** [comment]
I would like a meet monthly or bi-weekly, for ex matplotlib. Where some members can give basic intro to these trivial steps, basic code/repo walkthrough. And next meeting someone new can volunteer.
